tests/kafkatest/services/kafka/kafka.py

```diff
@@ -138,6 +138,45 @@ def start_minikdc(self, add_principals=""):
         else:
             self.minikdc = None
 
+    def start_some(self, nodes_to_start, create_topics=True, add_principals=""):
```

Comment:
Yes, I think it can be deleted now. It was added because it would be needed for the second item in [CPKAFKA-393](https://confluentinc.atlassian.net/browse/CPKAFKA-393).

But we have since decided that the second item can be rolled into this PR.. @benstopford will submit a patch for that. In which case we don't need this code. Will delete it.